### DISCUSSION TITLE
Missing 0.7.x on conda

### DISCUSSION THREAD

**Discussion by sbhasan** [jorgensd/dolfinx_mpc#93] [General]
Hi,
I install dolfinx/dolfinx_mpc through conda. I noticed that dolfinx_mpc is still 0.6.x on conda-forge. I would appreciate if it is upgraded to the latest version.
Kind regards,
Shakeeb

**Comment by jorgensd** [comment]
We have had some issues with mpi on conda: https://github.com/conda-forge/dolfinx_mpc-feedstock/pull/6
It will be present once we have been able to solve these issues.

**Comment by jorgensd** [comment]
v0.7.0 is being rolled out on conda as we speak: https://anaconda.org/conda-forge/dolfinx_mpc
https://github.com/conda-forge/dolfinx_mpc-feedstock/pull/6
